go/processor/floria/processor_test.go

```diff
@@ -477,16 +534,17 @@ func TestProcessor_SetUpAccessList(t *testing.T) {
 		},
 	}
 
-	for _, contract := range getPrecompiledAddresses(tosca.R09_Berlin) {
+	for _, contract := range getPrecompiledAddresses(tosca.R13_Cancun) {
 		context.EXPECT().AccessAccount(contract)
 	}
 	context.EXPECT().AccessAccount(sender)
 	context.EXPECT().AccessAccount(recipient)
 	context.EXPECT().AccessAccount(accessListAddress)
 	context.EXPECT().AccessStorage(accessListAddress, tosca.Key{1})
 	context.EXPECT().AccessStorage(accessListAddress, tosca.Key{2})
+	context.EXPECT().AccessAccount(coinbase)
 
-	setUpAccessList(transaction, context, tosca.R09_Berlin)
+	setUpAccessList(transaction, context, tosca.R13_Cancun, coinbase)
```

Comment:
It is not enough to use one revision to test the code. This is basically just adding code coverage, but it does not check that other revisions are handled correctly.